src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -80,12 +82,14 @@ $sidebarActivityFeedSelectedItemBorderWidth: 2px;
     }
 }
 
-.be .bcs-activity-feed .bcs-activity-feed-items-container .bcs-ActivityItem.bcs-is-focused .bcs-ActivityThread .bcs-ActivityCard {
+.be .bcs-activity-feed .bcs-activity-feed-items-container .bcs-ActivityItem.bcs-is-focused .bcs-ActivityThread .bcs-ActivityCard,
+.be .bcs-activity-feed .bcs-activity-feed-items-container .bcs-ActivityItem.bcs-is-focused .bcs-ActivityThread .bcs-Comment {
```

Comment:
There's a lot going on here. I would recommend just duplicating the css for now and keeping it separate. I'm willing to sacrifice some brevity for readability in this case especially sense the other stuff will eventually go away.

Reply:
This file will be deleted